hyp3_testing/templates/autorift_golden.json.j2

```diff
@@ -88,5 +88,45 @@
        ]
     },
     "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LC09_L1GT_215109_20220125_20220125_02_T2",
+        "LC09_L1GT_215109_20220210_20220210_02_T2"
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LE07_L1TP_063018_20040911_20200915_02_T1",
+        "LE07_L1TP_063018_20040810_20200915_02_T1"
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LT05",
+        "LT05"
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LT04",
+        "LT04"
```

Comment:
@forrestfwilliams @jacquelynsmale can you add one of the Landsat 4 pairs you've been working with to add L4 support?

Reply:
@jhkennedy @forrestfwilliams this one was a bit trickier to select an example. We haven't been working with a specific L4 example, and searching for pairs was a bit harder. I selected a pair from Antarctica with cloud cover < 50 for both scenes. I'm not 100% confident in the quality here, but it's a pair. 
LT04_L1GS_217110_19900115_20200916_02_T2 and LT04_L1GS_217110_19891230_20200916_02_T2.

Reply:
I did some more searching in the pair list and the explorer, and found a pair that has cloud cover under 5% and I verified it is in our pair list for L4. The pair name is: `LT04L1GS0501171990012901T2_LT04L1GS0501171989031501T2_03031_G0240V01_P002.nc`, which are the scenes LT04_L1GS_050117_19890315_20200916_02_T2 and LT04_L1GS_050117_19900129_20200916_02_T2.